src/main/webapp/script.js

```diff
@@ -46,27 +46,36 @@ function updateFeedLinkForCurrentUser() {
   link.setAttribute('href', 'feed.html?id=' + uid);
 }
 
-function updateMatchesLinkForCurrentUser() {
-  let uid = getCurrentUserId();
-  var link = document.getElementById("matchesLink");
+function updateMatchesLinkForCurrentUser(userid) {
+  const link = document.getElementById("matchesLink");
   link.setAttribute('href', 'matches.html?id=' + uid);
 }
 
 window.addEventListener("load", function(){
```

Comment:
nit: unrelated to this change, but can you add a comment above this explaining what it does?